pkg/server/auth/server.go

```diff
@@ -429,6 +436,35 @@ func (s *AuthServer) UserInfo() http.HandlerFunc {
 	}
 }
 
+func (s *AuthServer) Refresh(rw http.ResponseWriter, r *http.Request) error {
+	ctx := oidc.ClientContext(r.Context(), s.client)
+
+	refreshTokenCookie, err := r.Cookie(RefreshTokenCookieName)
+	if err != nil {
+		return errors.New("couldn't fetch refresh token from cookie")
```

Comment:
Is this an error?

Or is it just "not refreshed" ?

Reply:
This is a fair point if offline_access is not a requested scope.. 🤔

Reply:
A new method shouldAttemptRefresh that checks?
- refresh_token even exists
- oidc is enabled
- offline-scope is enabled
?

Reply:
Probably check for refresh_token?

Would we have a refresh_token if oidc was not enabled/offline-scope was not enabled?

Reply:
true, fair.